presto-hive/src/main/java/io/prestosql/plugin/hive/metastore/thrift/ThriftMetastoreUtil.java

```diff
@@ -971,10 +971,13 @@ private static OptionalDouble getAverageColumnLength(OptionalLong totalSizeInByt
         if (type.equals(BOOLEAN)) {
             return ImmutableSet.of(NUMBER_OF_NON_NULL_VALUES, NUMBER_OF_TRUE_VALUES);
         }
-        if (isNumericType(type) || type.equals(DATE) || type.equals(TIMESTAMP_MILLIS)) {
-            // TODO https://github.com/prestosql/presto/issues/37 support non-legacy TIMESTAMP
+        if (isNumericType(type) || type.equals(DATE)) {
             return ImmutableSet.of(MIN_VALUE, MAX_VALUE, NUMBER_OF_DISTINCT_VALUES, NUMBER_OF_NON_NULL_VALUES);
         }
+        if (type.equals(TIMESTAMP_MILLIS)) {
+            // TODO https://github.com/prestosql/presto/issues/37 support non-legacy TIMESTAMP
```

Comment:
This comment was vestigial, there is no longer such a thing "legacy" and "non-legacy" timestamp.